lib/components/Note/Note.tsx

```diff
@@ -0,0 +1,57 @@
+export interface NoteProps {
+  /** Title shown on the note */
+  title?: string;
+  /** Description shown on the note */
+  description?: string;
+  /** Icon shown on the note */
+  variant?: 'success' | 'warning' | 'error';
+  /** Callback fired on tap/click of the close button */
+  onCloseClick?: () => void;
+  /** Text for read more link */
+  readMoreText?: string;
+  /** Link to read more */
+  readMoreHref?: string;
```

Comment:
As this "readMore" seems to be only visible when in success state, maybe that should be told here.